### PR TITLE
Feature/implement upload UI

### PR DESCRIPTION
**Pull request by Honchar007**

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (tedi4t) [inline env:1]: You have forgotten .env

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "@aws-sdk/client-cognito-identity-provider": "^3.42.0",
     "@emotion/react": "^11.6.0",
     "@emotion/styled": "^11.6.0",
+    "@material-ui/core": "^4.12.3",
```
Comment (tedi4t) [inline package.json:25]: We don't use @material-ui, we use @mui-ui. It's a different versions of same framework

File: src/components/UploadComponent/MultipleFileUploadField.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  // id was added after the video being released to fix a bug
+  // Video with the bug -> https://youtube-2021-feb-multiple-file-upload-formik-bmvantunes.vercel.app/bug-report-SMC-Alpha-thank-you.mov
+  // Thank you for the bug report SMC Alpha - https://www.youtube.com/channel/UC9C4AlREWdLoKbiLNiZ7XEA
+  id: number;
+  file: File;
+  errors: FileError[];
+  url?: string;
+}
+
+const useStyles = makeStyles((theme) => ({
```
Comment (tedi4t) [inline src/components/UploadComponent/MultipleFileUploadField.tsx:25]: We don't write like that. We use styled-components for it. And we don't have theme! Rewrite it as in example below.
https://github.com/Intelligent-Cloud-System/ics-frontend/blob/dev/src/pages/Login/Login.styles.ts

File: src/components/UploadComponent/FileHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { Button, Grid } from '@material-ui/core';
```
Comment (tedi4t) [inline src/components/UploadComponent/FileHeader.tsx:1]: write components in forders

File: src/pages/UploadPage/UploadPage.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```
Comment (tedi4t) [inline src/pages/UploadPage/UploadPage.tsx:1]: Where is this page in routing?

**Review by tedi4t — COMMENTED** [review]
Look throuth login page implementation. We have to keep one style

**Review by tedi4t — COMMENTED** [review]
You have totally ignored prettier!

**Review by tedi4t — COMMENTED** [review]
project does not even start

**Review by tedi4t — COMMENTED** [review]
Remove warnings when starting project

**Review by MaksGovor — APPROVED** [review]

File: env [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+REACT_APP_CORE_URL=http://localhost:3000/
```
Comment (tedi4t) [inline env:2]: Please do something to delete this file from git storage

File: src/pages/UploadPage/SingleFileUpload/FileHeader.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import { Button } from '@material-ui/core';
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import InsertDriveFileIcon from '@material-ui/icons/InsertDriveFile';
+import CardContent from '@mui/material/CardContent';
```
Comment (tedi4t) [inline src/pages/UploadPage/SingleFileUpload/FileHeader.tsx:8]: Hey, we don't use @material-ui. We use @mui-ui. We were talking about it several times

File: src/pages/UploadPage/SingleFileUpload/SingleFileUploadWithProgress.tsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+	return (
+		<Grid item>
+			<FileHeader file={file} onDelete={onDelete} />
+			<LinearProgress variant='determinate' value={progress} />
+		</Grid>
+	);
+}
+
+function uploadFile(file: File, onProgress: (percentage: number) => void) {
+	const url = 'https://api.cloudinary.com/v1_1/demo/image/upload';
```
Comment (tedi4t) [inline src/pages/UploadPage/SingleFileUpload/SingleFileUploadWithProgress.tsx:36]: What is it?

File: src/pages/UploadPage/SingleFileUpload/UploadError.tsx [hunk 9]
```diff
@@ -0,0 +1,32 @@
+import { createStyles, LinearProgress, Typography, withStyles } from '@material-ui/core';
```
Comment (tedi4t) [inline src/pages/UploadPage/SingleFileUpload/UploadError.tsx:1]: We don't need it. We already have shared interface for all errors

File: src/pages/UploadPage/SingleFileUpload/SingleFileUploadWithProgress.tsx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+	const key = 'docs_upload_example_us_preset';
+
+	return new Promise<string>((res, rej) => {
```
Comment (tedi4t) [inline src/pages/UploadPage/SingleFileUpload/SingleFileUploadWithProgress.tsx:39]: Somebody else will be responsible for it. You are responsible just for design.

File: src/pages/UploadPage/UploadPage.styles.ts [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+		return '#ff1744';
+	}
+	if (props.isDragActive) {
+		return '#2196f3';
+	}
+	return '#eeeeee';
+};
+
+export const ContainerDropDown = styled.div`
+	height: 50vh;
```
Comment (tedi4t) [inline src/pages/UploadPage/UploadPage.styles.ts:17]: Use mui-ui Grid element together with attributes

**Review by MaksGovor — APPROVED** [review]

**Pull request merged**